new/LIT7101QeneAmh.xml

```diff
@@ -0,0 +1,100 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7101QeneAmh" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="am" xml:id="t1">እየደረስብኝ፡ ነው፡ በሰኔ፡ ፈረሰኛው፡ ሞት፡ ነፍሴን፡ ሳልጠቅማት፡ ወየው፡ ሰዎች፡ እያሉ፡ አዝማሪ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎyyadarasabbəññ naw ba-sane farasaññāw mot (Qǝne possibly of the mawaddǝs-type)</title>
+                <title xml:lang="en" corresp="#t1">The galloping death is reaching me in Sane, without me having benefitted my soul. Oh my woes, the ʾazmāri says…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+                <abstract>
+                    <p>Amharic <foreign xml:lang="gez">qǝne</foreign> without type indication in the manuscript. It has eight lines, meaning
+                        that it can be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> according to
+                        <bibl><ptr target="bm:Guidi1900Qene"/></bibl>.
+                        It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
```

Comment:
I've based all the attributions to Gabra Leul on "initials" like those in the upper left corner, but you are right that it's possible to understand them otherwise… The first qene "attributed" to him has a fuller note saying: "Gabra Leul's collection of qene" (in Swedish). What do you think, @DenisNosnitsin1970 , @nafisa-valieva , @eu-genia , and @CarstenHoffmannMarburg ? Is it better to take this as an authorship attribution or not? I initially understood as an authorship attribution

Reply:
If we read what is written in tigrinya, actually he is not the author, but an unnamed chera wata /azmari.  He might just remembered the poem and dictated it.

Reply:
Yes, I think it is reasonable to find a formula that makes it possible to see G.L. as either the author or the transmitter in this poem as well as in the others.

Reply:
I will made a suggestion and you can say what you think. Thank you for the input and ideas! :)